src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/NewAzureRmRecoveryServicesAsrReplicationProtectedItem.cs

```diff
@@ -223,8 +223,8 @@ public class NewAzureRmRecoveryServicesAsrReplicationProtectedItem : SiteRecover
         /// <summary>
         ///     Gets or sets the ID of the Azure storage account to replicate to.
         /// </summary>
-        [Parameter(ParameterSetName = ASRParameterSets.EnterpriseToAzure, Mandatory = true)]
```

Comment:
It is semantically a breaking change but  it is OK if the cmdlet cannot work without the `RecoveryAzureStorageAccountId` for this parameter set.

Reply:
@msJinLei, the cmdlet should work without  RecoveryAzureStorageAccountId  for replication to MD scenario. If the user gives both RecoveryAzureStorageAccountId   and UseManagedDisksForReplication, even the service doesn't allow.